jwst/ami/tests/test_ami_leastsqnrm.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from jwst.ami import leastsqnrm
+
+
+def test_weighted_operations():
+    img = np.arange(4, dtype='f4').reshape((2, 2))
```

Comment:
@rcooper295 any suggestion for improving these fake values? At the moment this test passes but produces warnings:
```
jwst/ami/tests/test_ami_leastsqnrm.py::test_weighted_operations
  /Users/bgraham/projects/230621_oifits/jwst/jwst/ami/leastsqnrm.py:500: RuntimeWarning: divide by zero encountered in divide
    weights = np.where(flatimg <= 1.0, 0.0, 1.0 / np.sqrt(variance))  # anand 2022 Jan

jwst/ami/tests/test_ami_leastsqnrm.py::test_weighted_operations
  /Users/bgraham/projects/230621_oifits/jwst/jwst/ami/leastsqnrm.py:516: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
  To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
    x, rss, rank, singvals = np.linalg.lstsq(Aw, bw)
```
The first one looks like a result of the fake values. The second looks like `lstsq` should receive another argument.

Reply:
Does this indicate a bug somewhere? Are you able to look into it? Your familiarity with the code will certainly help.

Reply:
I think it's just due to the zero in the test data. If I do `img = np.arange(1,5, dtype='f4').reshape((2, 2))` I don't get that warning anymore.